rpm/build/build.sh

```diff
@@ -13,7 +13,8 @@ cp "$WAR" $D/SOURCES/jenkins.war
 
 pushd $D
   mkdir -p BUILD RPMS SRPMS
-  rpmbuild -ba --define="_topdir $PWD" --define="_tmppath $PWD/tmp" --define="ver $VERSION" SPECS/jenkins.spec
+  rpmbuild -bs --define="_topdir $PWD" --define="_tmppath $PWD/tmp" --define="ver $VERSION" SPECS/jenkins.spec
+  mock --define="ver $VERSION" --resultdir=RPMS --root=epel-7-x86_64 SRPMS/*.rpm
```

Comment:
you'll either need to install this in the build or add it to the docker image:
https://github.com/jenkins-infra/docker-packaging/blob/main/Dockerfile

Reply:
It may be hard: I don't think mock is packaged on Ubuntu 18.04 nor if it works in a container, but you also won't have the right RPM macros present with rpmbuild. I'll need to think about this.

Reply:
https://packages.ubuntu.com/bionic/mock

Reply:
I have no clue if that mock is new enough. For now I've tried a hack. If it works, I can send a PR to docker-packaging.

Reply:
Ok, clearly that hack doesn't work. There's no sudo and the non-root user isn't allowed to install packages.